test/fixtures/test-runner/output/no_refs.snapshot

```diff
@@ -1,35 +1,23 @@
 TAP version 13
 # Subtest: does not keep event loop alive
     # Subtest: +does not keep event loop alive
-    not ok 1 - +does not keep event loop alive
+    ok 1 - +does not keep event loop alive
```

Comment:
nit: does this test make sense anymore?

Reply:
Yes, I think the test still makes sense. We still want to know how the test runner behaves in this scenario. The individual tests themselves still do not keep the event loop alive.